src/ORM/Table.php

```diff
@@ -2304,4 +2305,28 @@ public function __debugInfo()
             'connectionName' => $conn ? $conn->configName() : null
         ];
     }
+
+    /**
+     * Validates that a belongsToMany or hasMany assoc has at least one entry.
+     *
+     * This can be used to validate any other array as well.
+     *
+     * @param mixed $value
+     * @return boolean
+     */
+    public function validateCount($value, $expectedCount = 0, $operator = '>')
```

Comment:
It is ok to have this method in the request validation part, but it definitely does not belong to the Table class. It should be a method in the Validator / Validation class.

That being said, there is room for validating the amount of association links by executing queries. It should be done as an application rule, but that can wait for another PR.